sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala

```diff
@@ -139,7 +140,13 @@ object ReloadWorkflow {
       )
     IO.write(entryPoint, depsFileContent.show)
 
-    Webpack.run(entryPoint.absolutePath, bundleFile.absolutePath)(workingDir, logger)
+    customWebpackConfigFile match {
+      case Some(configFile) =>
+        Webpack.run("--config", configFile.getAbsolutePath, entryPoint.absolutePath, bundleFile.absolutePath)(workingDir, logger)
```

Comment:
Does that mean that webpack still uses the given `entryPoint` and `bundleFile` instead of using those given in the configuration file?